Dockerfile.osx

```diff
@@ -38,8 +38,10 @@ RUN git clone https://github.com/mono/mono --branch ${mono_version} --single-bra
     ln -s /usr/bin/mono /root/dependencies/mono/bin/mono && \
     ln -s /usr/bin/mono-sgen /root/dependencies/mono/bin/mono-sgen && \
     ln -sf /usr/lib/mono/* /root/dependencies/mono/lib/mono && \
-    cp -rvp /etc/mono /root/dependencies/mono/etc && \
```

Comment:
We were again copying the structure wrongly if `/root/dependencies/mono/etc` didn't exist, as the copied `mono` folder would become the `${prefix}/etc` folder instead of `${prefix}/etc/mono`.